File: workspace/pset6/sentimental/mario/less/mario.py
from cs50 import get_int

def main():
    while True:
        height = get_int("Height: ")
        width = height
        if height >= 1 and height <= 23:
            break

    for i in range(height):
        num_hashes = i + 1
        num_spaces = width - num_hashes

        print(" " * num_spaces, end="")
        print("#" * num_hashes)

if __name__ == "__main__":
    main()

File: pset1/mario/less/mario.c
#include<stdio.h>
#include<cs50.h>

void draw(char type, int times)
{
    for (int i = 0; i < times; i++)
    {
        printf("%c", type);
    }
}
int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 0 || height > 23);
    int nhash = 1;
    for (int i = 0; i < height; i++)
    {
        draw(' ', height - 1 - i);
        draw('#', nhash + i);
        printf("\n");
    }
}
